core/src/main/java/hudson/model/Computer.java

```diff
@@ -268,6 +269,18 @@ public Computer(Node node) {
     	return Collections.unmodifiableList(result);
     }
 
+    @SuppressWarnings({"ConstantConditions","deprecation"})
+    @SuppressFBWarnings("RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE")
+    @Override
+    public void addAction(@Nonnull Action a) {
+        if (a == null) {
+            throw new IllegalArgumentException("Action must be non-null");
+        }
+        super.getActions().add(a);
+    }
+
+    // TODO implement addOrReplaceAction, removeAction, removeActions, replaceActions
```

Comment:
These were introduced in #2608 so there is no compatibility issue: they would never have worked on `Computer`.